src/components/Button/index.tsx

```diff
@@ -158,6 +159,9 @@ function Button(
     const theme = useTheme();
     const styles = useThemeStyles();
     const isFocused = useIsFocused();
+    const activeElement = useActiveElement();
+    const roles: string[] = Object.values(CONST.ACCESSIBILITY_ROLE);
+    const shouldDisableEnterShortcut = roles.includes(activeElement?.role ?? '') && activeElement?.role !== CONST.ACCESSIBILITY_ROLE.TEXT;
```

Comment:
Name of const roles is confusing, let's rename it to accessibilityRoles or remove it like below

```suggestion
    const shouldDisableEnterShortcut = Object.values(CONST.ACCESSIBILITY_ROLE).includes(activeElement?.role ?? '') && activeElement?.role !== CONST.ACCESSIBILITY_ROLE.TEXT;
```